src/universal/types/constEnums.d.ts

```diff
@@ -41,3 +44,10 @@ export const enum Layout {
   LAYOUT_GRID = 8, // 1x
   ROW_GUTTER = 16 // 2x
 }
+
+// powers of 2 for cheap sense of scale
+export const enum Radius {
```

Comment:
if we go the const enum route for border radii i'd opt for having a number in the name Radius.x2, Radius.x4, etc. There may come a day when we have a 6px border radius